Jenkinsfile

```diff
@@ -629,10 +618,10 @@ pipeline {
   }
 
   triggers {
-    parameterizedCron((scheduleTriggerJobName() == env.JOB_NAME) ? """
+    parameterizedCron(branchPatternCron().matcher(env.BRANCH_NAME).matches() ? """
```

Comment:
Trying to make this more consistent with the other drivers.  This usage also seems to make more sense logically.